public/app/plugins/datasource/stackdriver/datasource.ts

```diff
@@ -197,7 +200,7 @@ export default class StackdriverDatasource extends DataSourceApi<StackdriverQuer
     let status, message;
     const defaultErrorMessage = 'Cannot connect to Stackdriver API';
     try {
-      const projectName = await this.getDefaultProject();
+      const projectName = await this.projectName;
```

Comment:
projectName is not a promise so not sure why this is awaited?

Reply:
Indeed, fix !